init_sd_card/stacks/daffy/DT18_03_roscore.yaml

```diff
@@ -2,7 +2,17 @@ version: '3'
 services:
   
   duckiebot-interface:
-    image: duckietown/dt-duckiebot-interface:devel20
+    image: duckietown/dt-duckiebot-interface:daffy
+    restart: unless-stopped
+    network_mode: host
+    privileged: true
+    environment:
+      DISABLE_CONTRACTS: 1
+    volumes:
+    - /data:/data
+
+  duckiebot-interface:
+    image: duckietown/dt-car-interface:daffy
     restart: unless-stopped
     network_mode: host
     privileged: true
```

Comment:
We should still mount `/data`, that is where `dagu_car` takes the calibration from, right?

Reply:
it is still mounted

Reply:
what I meant is that we need 
```
volumes:
    - /data:/data
```
right?